package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "parse": "node index.js",
+    "parse": "node App.js",
     "start": "nodemon index.js"
```

Comment:
You can remove the line 8, with the `start` script

Reply:
Done removing the start script